tools/export/gnuarmeclipse/__init__.py

```diff
@@ -1014,8 +1014,7 @@ def process_options(self, opts, flags_in):
         opts['cpp']['other'] += ' ' + \
             ' '.join(flags['common_flags']) + ' ' + \
             ' '.join(flags['cxx_flags'])
-        opts['ld']['other'] += ' ' + \
-            ' '.join(flags['common_flags']) + ' ' + ' '.join(flags['ld_flags'])
+        opts['ld']['other'] += ' ' +  ' '.join(flags['ld_flags'])
```

Comment:
I'll just take that commit off the top then. There does not seem to be a problem with adding the common flags. I'm just worried that any differences in the future will create failing exports if we are not careful.